test/11_array.jl

```diff
@@ -14,6 +14,7 @@ using CategoricalArrays: DefaultRefType, leveltype
     @test leveltype(x) === String
     @test eltype(x) === CategoricalValue{String, R}
     @test isordered(x) === ordered
+    @test isordered(x[1]) === ordered
```

Comment:
This file tests arrays, not values. Please add tests e.g. to 10_isless.jl instead.